apps/website/src/pages/docs/anchor.mdx

```diff
@@ -9,19 +9,37 @@ thumbnail: Anchor
 
 import PropsTable from '~/components/PropsTable.astro';
 import LiveExample from '~/components/LiveExample.astro';
-import Placeholder from '~/components/Placeholder.astro';
+import RelatedComponents from '~/components/RelatedComponents.astro';
 import * as AllExamples from '~/examples';
 
 <p>{frontmatter.description}</p>
 
-<Placeholder componentPageName='typography-anchor' />
-
 <LiveExample src='Anchor.main.tsx' truncate={false}>
   <AllExamples.AnchorMainExample client:load />
 </LiveExample>
 
+The anchor's `href` attribute is used to indicate the link's destination while the text within the anchor element should describe the link's destination.
+
+## Usage
+
 Consider using a named anchor when you want the user to land not only on a specific page, but also to arrive at a content area that may be below the fold so the user does not have to scroll to discover it.
 
+A common question is whether to use an anchor or a [button](button). The basic rule of thumb is an anchor takes you to another page, whereas a button performs an action. [Read this article to learn more about when to use a button vs an anchor.](https://yatil.net/blog/buttons-vs-links)
+
+### `as` prop
+
+The anchor component supports the polymorphic `as` prop to render the anchor as a button.
```

Comment:
More context can be added.
```suggestion
The anchor component supports the polymorphic `as` prop to render the anchor as a button. This can be useful when the design requires a control which visually looks like an anchor but acts like a button, i.e. it performs an action within the same page, rather than navigating to a destination URL.
```

Reply:
Added